test/modules/TestResultCoordinator/Settings.cs

```diff
@@ -143,6 +143,8 @@ List<IReportMetadata> InitializeReportMetadataList()
                 new CountingReportMetadata("relayer2.send", "relayer2.eventHub", TestOperationResultType.Messages, TestReportType.CountingReport),
                 new CountingReportMetadata("directMethodSender1.send", "directMethodReceiver1.receive", TestOperationResultType.DirectMethod, TestReportType.CountingReport),
```

Comment:
please comment out the other 2 direct method reports as well.

Reply:
yessir